tests/models/test_main.py

```diff
@@ -1176,6 +1182,30 @@ def test_cannot_remove_award_from_brief_response_if_brief_awarded(self):
 
         assert 'Cannot remove or change award datestamp on previously awarded Brief Response' in e.value.message
 
+    @pytest.mark.parametrize('awarded_at, expected_result_count', [
+        (datetime(2016, 3, 16, 23, 59, 59), 0),
+        (datetime(2016, 3, 17, 0, 0, 0), 1),
+        (datetime(2016, 3, 17, 23, 59, 59), 1),
+        (datetime(2016, 3, 18, 0, 0, 0), 0),
+    ])
+    def test_query_brief_response_returns_awarded_at_between_two_datetimes(self, awarded_at, expected_result_count):
```

Comment:
And this one I think we should rename `test_query_brief_responses_awarded_within_date_range`.